### PR TITLE
Falcon4 and OVC3 components Added

### PR DESCRIPTION
**Pull request by ackoc23**
Falcon4 and OVC3 components Added

### CODE REVIEW

File: mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+    <xacro:property name="M_PI" value="3.1415926535897931" />
+
+    <xacro:macro name="range_beam_model" params="name parent *origin">
+      <joint name="${name}_joint" type="fixed">
+        <xacro:insert_block name="origin" />
+        <parent link="${parent}"/>
+        <child link="${name}_frame"/>
```
Comment (tdinesh) [inline mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro:10]: remove extra `_frame`

File: mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      </joint>
+
+      <link name="${name}_frame">
```
Comment (tdinesh) [inline mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro:13]: remove extra `_frame`

File: mrsl_quadrotor_description/urdf/sensors/range_beam.urdf.xacro [hunk 3]
```diff
@@ -19,7 +19,7 @@
         <visual>
           <origin xyz="0 0 0" rpy="0 ${M_PI/2} 0" />
           <geometry>
-            <cylinder radius="0.01" length="0.1"/>
+            <cylinder radius="0.01" length="0.01"/>
```
Comment (tdinesh) [inline mrsl_quadrotor_description/urdf/sensors/range_beam.urdf.xacro:22]: remove extra `_frame` from this file line 10, 13, 32

File: mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro (hunk 1 above)
Comment (ackoc23) [reply mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro:10]: I removed the lidarlite.urdf.xacro file 
This is a file I added at first and than didn't use it

File: mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro (hunk 2 above)
Comment (ackoc23) [reply mrsl_quadrotor_description/urdf/sensors/lidarlite.urdf.xacro:13]: I removed the lidarlite.urdf.xacro file 
This is a file I added at first and than didn't use it

File: mrsl_quadrotor_description/urdf/sensors/range_beam.urdf.xacro (hunk 3 above)
Comment (ackoc23) [reply mrsl_quadrotor_description/urdf/sensors/range_beam.urdf.xacro:22]: I removed the extra _frame from lines 10, 13, 32
I have tested the code after removing and it works fine
I have pushed after the testing

**Pull request merged**